src/generateTeam.js

```diff
@@ -0,0 +1,141 @@
+const Manager = require("../lib/Manager");
+const Engineer = require("../lib/Engineer");
+const Intern = require("../lib/Intern");
+
+// function that generates card for team (Manager and/or Engineer and/or Intern)
+function generateTeamCards(team) {
+  let cards = [];
+  // loop through team members and generate relevant card for each
+  for (let i = 0; i < team.length; i++) {
+    const teamMember = team[i];
+    switch (teamMember.getRole()) {
+      case "Manager":
+        const manager = new Manager(
```

Comment:
Because your team member is already an instance of the manager you could simply do:

```
card.push(teamMember)
```

Right now you're creating an identical instance of a manager as the one you already have which doesn't really benefit your code.